### PR TITLE
fixes PLP/PLE execution

### PR DESCRIPTION
**Pull request by wivern**
fixes execution for PLP/PLE analyses

### CODE REVIEW

**Comment by pavgra** [comment]
@wivern , could you verify that upgrade from 2.5.2 (2.5.3) to 2.7.0 won't fail after the PR?

**Comment by wivern** [comment]
> @wivern , could you verify that upgrade from 2.5.2 (2.5.3) to 2.7.0 won't fail after the PR?

I've checked upgrade to 2.7 with Sql Server and Oracle, it's ok with #844

File: src/main/java/org/ohdsi/webapi/executionengine/controller/ScriptExecutionController.java [hunk 1]
```diff
@@ -52,6 +46,8 @@
 public class ScriptExecutionController {
 
     private final Log logger = LogFactory.getLog(ScriptExecutionController.class);
+    private static final String DATASOURCE_PERMISSION = "source:%s:get";
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/executionengine/controller/ScriptExecutionController.java]: Invalid permission for the case

File: src/main/java/org/ohdsi/webapi/executionengine/controller/ScriptExecutionController.java [hunk 2]
```diff
@@ -97,6 +95,10 @@ public JobExecutionResource runScript(ExecutionRequestDTO dto) {
         logger.info("Received an execution script to run");
         Source source = sourceRepository.findBySourceKey(dto.sourceKey);
 
+        if (securityEnabled && !SecurityUtils.getSubject().isPermitted(String.format(DATASOURCE_PERMISSION, source.getSourceKey()))) {
+            throw new ForbiddenException(String.format(FORBIDDEN_MESSAGE, source.getSourceKey()));
```
Comment (pavgra) [inline src/main/java/org/ohdsi/webapi/executionengine/controller/ScriptExecutionController.java]: Throws 500 instead of 403

**Pull request merged**